### PR TITLE
Add update-image, deploy and undeploy functions

### PR DESCRIPTION
**Pull request by poupotte**
* **update-image**: Destroys and recreates from new image
* **deploy**: Allows to access to application via the box : 
    * Add in database
    * Reset proxy
    * Forward port
* **undeploy**: "Uninstall" application from box

### CODE REVIEW

File: src/main.coffee [hunk 1]
```diff
@@ -99,19 +99,35 @@ program
...
+        port = 9250
+    # Install application for cozy stack
+    projectManager.recoverManifest port, (err, manifest) ->
+        return console.log err if err?
+        appManager.addInDatabase manifest, (err) ->
+            return console.log err if err?
+            appManager.resetProxy (err) ->
+                return console.log err if err?
+                appManager.addPortForwarding manifest.name, port, (err) ->
+                    return console.log err if err?
```
Comment (jsilvestre) [inline src/main.coffee]: `async.waterfall` could improve this piece of code

File: src/main.coffee [hunk 2]
```diff
@@ -99,19 +99,35 @@ program
...
+.action () ->
+    # Uninstall application for cozy stack
+    projectManager.recoverManifest 9250, (err, manifest) ->
+        return console.log err if err?
+        appManager.removeFromDatabase manifest, (err, name, port) ->
+            return console.log err if err?
+            appManager.resetProxy (err) ->
+                return console.log err if err?
+                appManager.removePortForwarding name, port, (err) ->
+                    return console.log err if err?
```
Comment (jsilvestre) [inline src/main.coffee]: same thing, `async.waterfall` could help

**Comment by jsilvestre** [comment]
Those features are awesome.

File: src/main.coffee (hunk 2 above)
Comment (frankrousseau) [inline src/main.coffee]: I would say sub-functions could improve that piece of code.

File: src/main.coffee (hunk 1 above)
Comment (frankrousseau) [inline src/main.coffee]: I would say sub-functions could improve that piece of code.

File: src/main.coffee (hunk 2 above)
Comment (jsilvestre) [inline src/main.coffee]: There are already sub functions for each step.

File: src/main.coffee (hunk 1 above)
Comment (jsilvestre) [inline src/main.coffee]: There are already sub functions for each step.

**Comment by frankrousseau** [comment]
Thx @poupotte !

**Pull request merged**